src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, jobID=None, jobReport=None):
             self.jobReport = JobReport(self.jobID, "JobWrapper@%s" % self.siteName)
         self.failoverTransfer = FailoverTransfer()
 
-        self.log = gLogger.getSubLogger("JobWrapper[%s]" % self.jobID)
+        self.log = gLogger.getSubLogger(__name__ + f"[{self.jobID}]")
```

Comment:
I think we better have the same format everywhere:

```suggestion
        self.log = gLogger.getSubLogger(f"[{self.jobID}]{__name__}")
```